### PR TITLE
aXe accessibility audit issues, part 2

### PR DESCRIPTION
**Pull request by wkillerud**
Part 2 for #163 
Related to #164 

There are still some issues. Color contrasts, duplicate IDs. But this is already a monster of a PR :crying_cat_face:

### CODE REVIEW

File: packages/ffe-account-selector-react/src/components/account-selector-multi/AccountSelectorMulti.md [hunk 1]
```diff
@@ -6,49 +6,53 @@ initialState = {
...
+                });
+            } else {
+                setState({
+                    selectedAccounts: [
+                        ...selectedAccounts,
+                        acc,
+                    ],
+                });
+            }
+        }}
```
Comment (selbekk) [inline packages/ffe-account-selector-react/src/components/account-selector-multi/AccountSelectorMulti.md:52]: Could it be an idea to pull these long functions and data structures out of the actual JSX and place them above?  This goes for all examples.

File: packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md [hunk 2]
```diff
@@ -1,47 +1,54 @@
-The default:
+En komponent for å kunne søke igjennom en lenger liste med valg. Listen med valg har støtte for å vise flere atributter ved et element, for eksempel bedriftnavn og organisasjonsnummer.
```
Comment (selbekk) [inline packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md]: attributter*

File: packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md [hunk 3]
```diff
@@ -1,47 +1,54 @@
...
+
+Om du vet du har 10 valg eller færre er det bedre å bruke en vanlig [dropdown](#dropdown).
+
+Om du er ute etter en måte å vise kontoinformasjon finnes en skreddersydd [kontovelger](#kontovelger).
```
Comment (selbekk) [inline packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md]: An aside - but should we use absolute URLs? This won't work when you're viewing a details page.

File: packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md (hunk 2 above)
Comment (selbekk) [reply packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md]: Noticed the same typo a few other places too. Perhaps you can spell it both ways? 🤷‍♂️

**Review by selbekk — DISMISSED** [review]
Great job!

File: packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md (hunk 2 above)
Comment (wkillerud) [reply packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md]: Darnit, Google-as-a-spellchecker failed me :(

**Pull request merged**